vast/integration/vast_integration_suite.yaml

```diff
@@ -819,3 +819,40 @@ tests:
         input: data/json/empty-record.json
       - command: exec 'from - read json | write xsv to stdout'
         input: data/json/empty-record.json
+
+  Zeek TSV Pipeline Format:
+    tags: [pipelines, zeek]
+    steps:
+      - command: exec 'from stdin read zeek-tsv | write json to stdout'
+        input: data/zeek/merge.log
+      - command: exec 'from stdin read zeek-tsv | write json to stdout'
+        input: data/zeek/merge_with_whitespace_separation.log
+      - command: exec 'from stdin read zeek-tsv | write zeek-tsv to stdout'
+        input: data/zeek/dns.log.gz
+      - command: exec 'from stdin read zeek-tsv | write csv to stdout'
+        input: data/zeek/dns.log.gz
+      - command: exec 'from stdin read zeek-tsv | write zeek-tsv ; empty NULLVAL to stdout'
+        input: data/zeek/whitespace_start.log
+      - command: exec 'from stdin read json | write zeek-tsv to stdout'
+        input: data/json/snmp.log.json.gz
+      - command: exec 'from stdin read zeek-tsv | write zeek-tsv to stdout'
+        input: data/zeek/broken1.log
```

Comment:
Could you add comments that indicate _why_ these are broken? Hard to see from the input.

Reply:
I renamed the files to specify what exactly is broken in I reduced the expected events in https://github.com/tenzir/vast/pull/3114/commits/624e9d93cde83d8e69646712bc5458d69ae56b68